### ISSUE TITLE
WordPress DocBlock: Period at the end of the summary

### ISSUE THREAD

**Issue by grappler** [WordPress-Coding-Standards/WordPress-Coding-Standards#1098]
I was recently doing a review of a pull request for `_s`  where the period was missing at the end of the summary and WPCS was not picking up on it.

The other values in the DocBlock also require a period at the end of the line.

There is an existing sniff to check for inline comments `Squiz.Commenting.InlineComment.InvalidEndChar` but not for function DocBlocks.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#docblock-formatting

**Comment by GaryJones** [comment]
Just as a point of note, there are many aspects of WP Inline PHP Docs standards that we don't check for - it was just the existing sniffs from other standards that were added in. (#241, #381).

Anything else will require custom sniffs.

**Comment by jrfnl** [comment]
Yes, I've looked into what's covered and what's missing from the Docs standard before, but due to other things having higher priority not actioned it so far.

I have an inventarisation available verified by @drewapicture

There is a lot which we would want to add/change if we want to properly cover the WP documentation standards.

**Comment by killua99** [comment]
How could I support this kind of Comment Blocks.

`	/**
	 * {@inheritdoc}
	 */
`

I was trying WPCS: CSRF ok. but couldn't make it work.

**Comment by GaryJones** [comment]
> I have an inventarisation available verified by @DrewAPicture

@jrfnl Do you still have that? Could you share please?

**Comment by GaryJones** [comment]
> How could I support this kind of Comment Blocks.
> 
> /** * {@inheritdoc} */

While I don't think the use of `inheritdoc` is mentioned in the Handbook, I do think we should allow it. @killua99 please open a new issue with it, so it's not lost in this thread about summary periods :-)

**Comment by jrfnl** [comment]
@GaryJones I do still have it.

I think the Docs ruleset should be treated as a project and needs to be a coordinated effort. Sharing the specs without defining the project and without further explanation would IMHO be counter-productive.
The spreadsheet only contains the specs, not the outline of the sniff architecture which is needed.
